INSTALL

```diff
@@ -9,6 +9,20 @@ wiki pages:
 * If you are in a hurry *
 *************************
 
+On Linux (Debian):
+
+ Note: you will need to add the Debian 'unstable' distribution to your source
```

Comment:
Is there not also a way to add a PPA? Which is preferred in the debian world? How long will it be until it ends up in stable?

PS I am so excited about this.